HandmadeMath.h

```diff
@@ -1380,13 +1451,15 @@ HMM_EqualsVec3(hmm_vec3 Left, hmm_vec3 Right)
 HINLINE hmm_bool
 HMM_EqualsVec4(hmm_vec4 Left, hmm_vec4 Right)
 {
-    hmm_bool Result = 0;
+    hmm_bool Result;
 
     Result = (Left.X == Right.X && Left.Y == Right.Y && Left.Z == Right.Z && Left.W == Right.W);
 
     return (Result);
 }
 
+// TODO(zak): Remove before pushing to main.
+// I dont think its worth us SSEing HMM_Mat4 since its just a mem-zero
```

Comment:
@StrangeZak We still have these comments here, even though it appears we're not doing any SSE in here.